src/screens/SUE/SueMapScreen.tsx

```diff
@@ -165,6 +172,11 @@ export const SueMapScreen = ({ navigation, route }: Props) => {
 };
 
 const styles = StyleSheet.create({
+  closeButton: {
+    position: 'absolute',
+    right: normalize(0),
+    top: normalize(0)
```

Comment:
you do not need to normalize 0

Reply:
Yes you are right. But now I updated the `normalize(0)` code.

https://github.com/smart-village-solutions/smart-village-app-app/commit/5729d04a27b9b9836835c00479f7d4caf6b72829